components/Footer.tsx

```diff
@@ -1,12 +1,48 @@
-import TextImage from "@/TextImage";
-import img from "../public/favicon.jpg";
+import Image from "next/image";
+import img from "../public/footerLogo.svg";
+import paws from "../public/paws.svg";
 
 const Footer = () => {
 
-    return <footer className={`mt-10 flex flex-col justify-between items-center h-full w-full bg-black text-white`}>
-        Footer
-        <div className={`text-orange`}>broj za contact: +389123456789</div>
-        <TextImage image={img} text={"Placeholder"} style={`mt-10 flex flex-col justify-between items-center h-full w-full bg-black text-white`}/>
-    </footer>
+    return (
+        <footer className="w-full flex items-center justify-center pb-[76px] pt-[50px]">
+            <div className="flex flex-col items-center relative border-2 rounded-[25px] border-[#FFAC66] w-[1240px] h-[281px]">
+                <Image src={img} alt={"Hero Image"} width={104} height={84} className="absolute top-[30px]" />
+                <p className="absolute top-[122px] text-[17.33px] text-black">077 805 229</p>
+
+                <Image src={paws} alt={"Paws 1"} width={78.74} height={78.74} className="absolute top-[113px] left-[81px] rotate-[-157deg]"></Image>
```

Comment:
Image is a self-closing tag, so </Image> is unnecessary. 
Additionally when sourcing statically imported images, there's no need to provide width/height values.